packages/aws-cdk/lib/init-templates/v1/app/javascript/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Can you add a comment to this PR titled 'Testing' on how you tested this change? At a minimum, that the experience doesn't regress.

context: we don't have very good automated test coverage for our init templates, and have got it wrong many times.